taxonomies/food/ingredients.txt

```diff
@@ -85393,12 +85394,20 @@ ciqual_food_code:en: 20210
 ciqual_food_name:en: Cucumber, pulp, raw
 ciqual_food_name:fr: Concombre, pulpe, cru
 
-
-
 # comment:en:From wikidata it is not clear if a language differentiates between cucumber and gherkin
 
 < en:cucumber
-en: gherkin
+en: baby cucumber, baby cucumbers, small cucumber, raw gherkin, unpickled gherkin
+sv: smågurka, smågurkor, smågurk
+
+< en:cucumber
+en: pickled cucumber
+da: syltet agurk, syltede agurker
+wikidata:en: Q1365891
+
+< en:baby cucumber
+< en:pickled cucumber
+en: pickled gherkin, pickle
```

Comment:
This one should be removed right? We already have an entry "en: Pickled gherkins" above. Also "pickle" by itself should not be asynonym.

Reply:
`en: Pickled gherkins` is a category. `pickle` by itself is already a synonym, I just moved it: https://github.com/openfoodfacts/openfoodfacts-server/pull/11806/files#diff-bf065447ce91af597ab8343c4053a9c4c17d8f4409ec243012daf06aae7f38c4L85321